src/mnemonic/generateseed.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2019 The VEIL developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <mnemonic/generateseed.h>
+#include <key.h>
+#include <mnemonic/mnemonic.h>
+
+namespace navcoin {
+
+uint512 GenerateNewMnemonicSeed(std::string& mnemonic, const std::string& strLanguage)
+{
+    CKey key;
+    key.MakeNewKey(true);
+
+    std::vector<unsigned char> keyData;
```

Comment:
using this vector to store the key data is unsafe, as it will leave traces in memory of the seed.
an option would be to modify create_mnemonic to take directly a CKey object, which uses secure_allocator (src/support/allocators/secure.h) and ensures memory is clean when the object is destroyed

Reply:
This is just code from Veil you didn't pull to Navcoin when bringing over Mnemonic to Navcoin so I would really have no idea what is best.

Reply:
Ok that is fixed now